addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -127,13 +137,18 @@ function factory(dependencies) {
         date: attr({
             compute: '_computeDate',
         }),
-        id: attr({
+        notification_type: attr({
+            readonly: true,
+        }),
```

Comment:
Could be nice to have `camelCase`

Reply:
Out of scope? I just added the `readonly`, I don't want to look for them in code and risk not renaming one.

Reply:
fair enough